README.md

```diff
@@ -80,30 +96,6 @@ You can click [here](https://devcenter.heroku.com/articles/heroku-postgres-impor
 
 **We will work on creating a script that would do the setup automatically for you but for now, please follow these steps and let any of the collaborators know if you have any issues.**
 
```

Comment:
@andela-kanyanwu How come we still have this line even when the script is no longer available?